types/testing-library__user-event/index.d.ts

```diff
@@ -1,20 +1,18 @@
-// Type definitions for @testing-library/user-event 4.0
+// Type definitions for @testing-library/user-event 4.1
 // Project: https://github.com/testing-library/user-event
 // Definitions by: Wu Haotian <https://github.com/whtsky>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export interface UserOpts {
+export interface IUserOptions {
```

Comment:
any particular reason for changing the interface's name?

IMO it's a breaking change and the "I" prefix is making tslint unhappy

Reply:
Yes, sorry, on my computer it was complaining about the I prefix within my project. I have resolved it now :)

Regarding the name just felt better to me but reverted the name change